src/main/java/trackitnus/model/commons/Name.java

```diff
@@ -11,13 +11,13 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+        "Names should be a non-empty string and don't contain '/'";
```

Comment:
I think should change to `Names should be a non-empty string and should not contain '/'`

Reply:
Sure!